scripts/generate_feature_catalog_page.py

```diff
@@ -39,16 +37,16 @@ def __init__(self, file_path: str):
 
     def add_service_section(self, feature_file_content: str):
         service_name = feature_file_content.get('name')
-        emulation_level = feature_file_content.get('emulation_level', DEFAULT_EMULATION_LEVEL)
+        emulation_level = feature_file_content.get('emulation_level')
         self.md_content.append(f"| **{service_name}** | [Details 🔍] | {emulation_level} | |")
 
     def add_features_rows(self, feature_file_content: str):
         for feature in feature_file_content.get('features', []):
             feature_name = feature.get('name', '')
-            documentation_page = feature.get('documentation_page')
-            if documentation_page:
-                feature_name = f'[{feature_name}]({documentation_page})'
-            status = feature.get('status', DEFAULT_STATUS)
+            aws_docs_url = feature.get('aws_documentation_url')
+            if aws_docs_url:
```

Comment:
comment: Since you've made it required in the schema you theoretically won't need this check in there anymore. But I guess it's good to stay defensive :D